### PR TITLE
Update ug

### PR DESCRIPTION
**Pull request by LeeMingDe**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -63,19 +65,6 @@ Covigent is a desktop app for managing information of quarantined individuals an
...
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-
-
-
```
Comment (itssodium) [inline docs/UserGuide.md:76]: Can delete white space, seems unnecessary.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -204,14 +139,7 @@ Covigent data are saved in the hard disk automatically after any command that ch
...
-**Add Rooms** | `addRooms NUMBER_OF_ROOMS` <br> e.g., `addRooms 123`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit Patient** | `editpatient NAME [n/NAME] [t/TEMPERATURE] [d/PERIOD_OF_STAY] [p/PHONE_NUMBER] [a/AGE] [c/COMMENT]`<br> e.g.,`editpatient James Lee t/36.5`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
+**Add Rooms** | `addRooms NUMBER_OF_ROOMS` <br> e.g., `addRooms 123`
 **Find Room** | `findRoom` <br> e.g `findRoom`
-**List** | `list`
-**Help** | `help`
-=======
```
Comment (itssodium) [inline docs/UserGuide.md:215]: Delete this line, I think its from a merge conflict.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -139,57 +98,33 @@ Examples:
...
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+### 3.4 Add rooms in hotel: `addRooms`
 
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-
```
Comment (itssodium) [inline docs/UserGuide.md:155]: Delete this single white space.

**Review by itssodium — APPROVED** [review]
Just make the little changes, other than that lgtm! Good job!

File: docs/UserGuide.md (hunk 1 above)
Comment (LeeMingDe) [reply docs/UserGuide.md:76]: the - means that its deleted

File: docs/UserGuide.md (hunk 2 above)
Comment (LeeMingDe) [reply docs/UserGuide.md:215]: all the red lines are deleted lines!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -11,7 +11,9 @@ title: User Guide
     3.1 [Command Format](#31-command-format)<br>
     3.2 [Add a patient: `addpatient`](#32-add-a-patient-addpatient)<br>
     3.3 [Edit a patient: `editpatient`](#33-edit-a-patient-editpatient)<br>
-    3.4 [Save the data](#34-save-the-data)<br>
+    3.4 [Add rooms in hotel: `addRooms`](#34-add-rooms-in-hotel-addrooms)<br>
+    3.5 [Finds the next room which is free to use: `findRoom`](#35-finds-the-next-room-which-is-free-to-use-findroom)<br>
```
Comment (raymondge) [inline docs/UserGuide.md]: I think "next" could be a little vague, coz the user may not know which room they are in, maybe "finds the first free room" could be better?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -139,57 +98,33 @@ Examples:
...
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
- 
-
-### Deleting a person : `delete`
-
-Deletes the specified person from the address book.
+Adds the number of room in a hotel
```
Comment (raymondge) [inline docs/UserGuide.md]: just minor gramma, "rooms"

**Review by raymondge — CHANGES_REQUESTED** [review]
LGTM, but just some minor error

File: docs/UserGuide.md (hunk 5 above)
Comment (LeeMingDe) [reply docs/UserGuide.md]: okay, will change it, thanks!

File: docs/UserGuide.md (hunk 4 above)
Comment (LeeMingDe) [reply docs/UserGuide.md]: okay can, I will change it

**Review by raymondge — APPROVED** [review]
LGTM!

**Pull request merged**